screenshot-test/data/api-companies.json

```diff
@@ -10,7 +10,18 @@
     "businessContact": "Representative",
     "legalContact": "override orchid reboot",
     "companyDomains": ["oncokb.org"],
-    "numberOfUsers": 0
+    "numberOfUsers": 0,
+    "additionalInfo": {
+      "license": {
+        "termination": {
+          "date": "2024-09-21",
```

Comment:
I know this is a fake data, but can we have termination date after activation date? 😃